miner/worker.go

```diff
@@ -268,6 +271,15 @@ func (w *worker) createCurrentEnvironment(predicateContext *precompileconfig.Pre
 	if err != nil {
 		return nil, fmt.Errorf("calculating gas capacity: %w", err)
 	}
+	// If Fortuna has activated, enforce we only build a block when there is a minimum
+	// built up gas capacity.
+	if chainConfigExtra.IsFortuna(header.Time) {
+		capacityPerSecond := header.GasLimit / acp176.TimeToFillCapacity
```

Comment:
can we have some comments why we dont use a constant here

Reply:
Good point - we only really want to ensure that large transactions up to existing expectations are able to get in. This means that there's little to gain from ever going above 15M even if the target increases in the future.

Reply:
Added a comment explaining this logic https://github.com/ava-labs/coreth/pull/995/commits/eed1b56078cccae791113b5d9200c89faccb9fe5